src/components/TestsideInformasjon.tsx

```diff
@@ -3,29 +3,39 @@ import Veilederpanel from 'nav-frontend-veilederpanel';
 import svg from '../assets/VeilederSvg';
 import { Panel } from 'nav-frontend-paneler';
 import Environment from '../Environment';
+import styled from 'styled-components';
+
+const StyledTestside = styled.div`
```

Comment:
Hører gjerne litt om erfaringene deres med styled components hvis dere går 100% for det.

Reply:
Vi ble enig i teamet om å teste det. 
Jeg er ikke super dreven i det, men hittil har jeg bare brukt det for de komponentene som ikke har så mye css. Det har funket bra :)